modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -0,0 +1,224 @@
++
+.. After review, run the `aws` commands manually to create the roles and policies. Alternatively, you can run the preceding command again using `--mode auto` to run the `aws` commands immediately.
+
+. Create a cluster by using the default installation options or by interactively specifying custom settings. To view other options when creating a cluster, enter `rosa create cluster --help`.
++
+[NOTE]
+====
+Multiple availability zones (Multi-AZ) are recommended for production workloads. The default is a single availability zone. Use `--help` for an example of how to set this option manually or use interactive mode to be prompted for this setting.
+====
++
+* To create a cluster with STS using the default installation options:
++
+[source,terminal]
+----
+$ rosa create cluster --cluster-name <cluster_name> --sts <1>
+----
+<1> Replace `<cluster_name>` with the name of your cluster.
++
+.Example output
+[source,terminal]
+----
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role for the Installer role
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role for the ControlPlane role
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role for the Worker role
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role for the Support role
+I: Creating cluster '<cluster_name>'
+I: To view a list of clusters and their status, run 'rosa list clusters'
+I: Cluster '<cluster_name>' has been created.
+I: Once the cluster is installed you will need to add an Identity Provider before you can login into the cluster. See 'rosa create idp --help' for more information.
+I: To determine when your cluster is Ready, run 'rosa describe cluster -c <cluster_name>'.
+I: To watch your cluster installation logs, run 'rosa logs install -c <cluster_name> --watch'.
+...
+----
++
+[NOTE]
+====
+If more than one matching set of account-wide roles are available in your account for a cluster version, an interactive list of options is provided.
+====
++
+* To create a cluster with STS using custom installation options:
++
+[source,terminal]
+----
+$ rosa create cluster --interactive
+----
++
+.Example output
+[source,terminal]
+----
+I: Interactive mode enabled.
+Any optional fields can be left empty and a default will be selected.
+? Cluster name: <cluster_name>
+? Deploy cluster using AWS STS: Yes
+? OpenShift version: 4.8.9
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Installer-Role for the Installer role
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-ControlPlane-Role for the ControlPlane role
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Worker-Role for the Worker role
+I: Using arn:aws:iam::<aws_account_id>:role/ManagedOpenShift-Support-Role for the Support role
+? External ID (optional): 
+? Operator roles prefix: <cluster_name>-z9y3
```

Comment:
Maybe call out that these are random characters: `<cluster_name>-<random_characters>`

Reply:
Thank you. Resolved.